rolling/distribution.yaml

```diff
@@ -5177,16 +5177,6 @@ repositories:
       type: git
       url: https://github.com/ros2/ros1_bridge.git
       version: master
-    release:
-      tags:
-        release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/ros1_bridge-release.git
-    source:
-      test_commits: false
-      type: git
-      url: https://github.com/ros2/ros1_bridge.git
-      version: master
-    status: maintained
```

Comment:
This is actually not quite what I intended. The doc stanza has been retained (but I'm not sure we can build documentation for ros1_bridge without ROS 1 nor does it make sense to me to provide docs for a package we don't make available and the source stanza has been removed.

What I think should remain is:
```yaml
  ros1_bridge:
    source:
      test_commits: false
      type: git
      url: https://github.com/ros2/ros1_bridge.git
      version: master
    status: <could we put something here about the deprecation due to no ROS 1 build?>
```

Reply:
> ```yaml
> 
>     status: <could we put something here about the deprecation due to no ROS 1 build?>
> ```

Apparently we have both `status`, which has to be one of `developed`, `maintained`, `unmaintained`, and `end-of-life`, as well as `status_description`, which is a free-form field (see https://www.ros.org/reps/rep-0141.html#distribution-file).  So I'm going to add in a `status_description` here, saying something about why this is not released.